functions/packages.sh

```diff
@@ -460,20 +474,19 @@ influxdb_grafana_setup() {
         if [ $? = 1 ]; then echo "CANCELED"; return 0; fi
       fi
       # Influx DB server address
-      local influxdb_returncode=0
-      local text_influxdb_address="Enter InfluxDB instance adress: [protocol:address:port] \n eg. https://192.168.1.100:8086"
+      influxdb_returncode=0
+      text_influxdb_address="Enter InfluxDB instance adress: [protocol:address:port] \n eg. https://192.168.1.100:8086"
       while [ "$influxdb_returncode" != "204" ]
       do
         influxdb_address=$(whiptail --title "InfluxDB" --inputbox "$text_influxdb_address" 15 80 3>&1 1>&2 2>&3)
         if [ $? = 1 ]; then echo "CANCELED"; return 0; fi
-        influxdb_returncode=`curl -s -o --max-time 6 --insecure /dev/null -w "%{http_code}" "$influxdb_address"/ping | sed 's/^0*//'`
+        influxdb_returncode="$(curl -s -o --max-time 6 --insecure /dev/null -w "%{http_code}" "$influxdb_address"/ping | sed 's/^0*//')"
```

Comment:
Looks as if this should be valid, just wanted to make sure you've tested it?

Reply:
Well i did use this branch to install an openhab instance on an APU2 (amd64) with grafana/influx.

As mentioned in https://github.com/openhab/openhabian/issues/498#issuecomment-453298772 I'm not sure if the way i tested it (i just used it) I was able to test/catch every change.

Reply:
I think this line is complicated to read because of multiple quotes. It should however execute correctly because of operator priority.